src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt

```diff
@@ -82,7 +83,7 @@ class MarkdownProjectReader(private val isHelp: Boolean) : IProjectReader {
     }
 
     private fun fileToId(f: OtterFile): Int =
-        f.nameWithoutExtension.toIntOrNull() ?: 0
+        f.nameWithoutExtension.toIntOrNull() ?: 1
```

Comment:
I would expect an id value of 0 means it hasn't been inserted into a database, wouldn't 1 cause problems?